Solutions/AristaAwakeSecurity/Analytic Rules/HighSeverityMatchesByDevice.yaml

```diff
@@ -0,0 +1,62 @@
+id: d5e012c2-29ba-4a02-a813-37b928aafe2d
+name: Awake Security - High Severity Matches By Device
+description: This query searches for devices with high severity event(s).
+severity: Medium
+requiredDataConnectors:
+  - connectorId: AristaAwakeSecurity
+    dataTypes:
```

Comment:
If the detection or hunting query operates on a Kusto Function/ Parser instead of the table (like Syslog, CommonEventFormat, _CL), dataTypes should be the Kusto Function name / Parser name and not the table name. However, looking at the query it seems that we are using just the table CommonSecurityLog and not some parser. Please correct me if I am mistaken here. If we are just using the table the dataTypes value should generally be the tablename i.e : CommonSecurityLog instead of AwakeSecurity